docs/sources/installation/SUSE.md

```diff
@@ -28,7 +28,7 @@ Docker is available in **SUSE Linux Enterprise 12 and later**. Please note that
 due to its current limitations Docker is able to run only on **64 bit**
 architecture.
 
-# Installation
+## Installation
```

Comment:
Ideally, we should be consistent with tense, etc. in headings. "Install Docker" "Uninstall Docker" or "Installation" "Uninstallation", etc.